File: c/bee1064.c
#include <stdio.h>
 
int main() {
    double v1, v2, v3, v4, v5, v6, media;
    scanf("%lf\n%lf\n%lf\n%lf\n%lf\n%lf", &v1, &v2, &v3, &v4, &v5, &v6);
    double valores[6] = {v1, v2, v3, v4, v5, v6};
    
    int cont = 0;
    double soma = 0;
    for(int i=0; i<6;i++){
        if(valores[i]>0){
            cont++;
            soma += valores[i];
        }
    }
    media = soma/cont;

    printf("%d valores positivos\n", cont);
    printf("%.1lf\n", media);


    return 0;
}

File: repos/beecrowd/python/bee1064.py
soma = pos = cont = 0
while True:
    num = float(input())
    if num != 0:
        if num > 0:
            pos += 1
            soma += num
        cont += 1
    if cont == 6:
        break
print(f'{pos} valores positivos')
print(f'{soma/pos:.1f}')
